Excercise 2.py

```diff
@@ -1,9 +1,10 @@
 type = (input("Are you converting celcius to fahrenheit? "))
-degrees = int(input("Give me the temperature"))
+degrees = int(input("Give me the temperature "))
 
 def converter(x, z):
```

Comment:
function names should be meaningfull also. 
What is this converter? What does it convert?